extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -91,6 +96,16 @@ export class SchemaCompareMainWindow {
 				packageFilePath: '',
 				connectionDetails: undefined
 			};
+		} else if (sourceDacpac) {
```

Comment:
in favor of CC - can we add a test case for checking endpoint creation for different inputs.. not sure if calling start() in UT will be possible but something to check

Reply:
added tests for the 3 different contexts schema compare can start with